CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,64 @@
+
+Violations of this code may result in swift and permanent expulsion from the Passage GitHub Community.
+
+# Scope
+We expect all members of the Passage GitHub Community — including administrators, users, facilitators, and vendors — to abide by this Code of Conduct at all times. This Code of Conduct applies in all 1Password Developer community venues, both online and in person, including all public and private channels and one-on-one communications pertaining to 1Password Developer affairs.
+
+This policy covers the usage of the Passage GitHub Community as well as both online and in-person events.
+
+This Code of Conduct is in addition to, and does not in any way nullify or invalidate, any other terms or conditions related to use of the Passage GitHub Community.
+
+The definitions of various subjective terms such as "discriminatory", "hateful", or "confusing" will be decided at the sole discretion of the 1Password Developer community administrators.
+
+# Friendly, Harassment-Free Space
+We are committed to providing a friendly, safe, and welcoming environment for all, regardless of gender identity, sexual orientation, disability, ethnicity, religion, age, physical appearance, body size, race, or similar personal characteristics.
+
+We ask that you respect that people have differences of opinion regarding technical choices, and acknowledge that every design or implementation choice carries a trade-off and numerous costs. There is seldom a single right answer. A difference of technology preferences is never a license to be rude.
+
+Any spamming, trolling, flaming, baiting, or other attention-stealing behavior is not welcome, and will not be tolerated.
+
+Harassing other users of the Passage GitHub Community is never tolerated, whether via public or private media.
+
+Avoid using offensive or harassing package names, nicknames, or other identifiers that might detract from a friendly, safe, and welcoming environment for all. Harassment includes, but is not limited to: harmful or prejudicial verbal or written comments related to gender identity, sexual orientation, disability, ethnicity, religion, age, physical appearance, body size, race, or similar personal characteristics; inappropriate use of nudity, sexual images, and/or sexually explicit language in public spaces; threats of physical or non-physical harm; deliberate intimidation, stalking or following; harassing photography or recording; sustained disruption of talks or other events; inappropriate physical contact; and unwelcome sexual attention.
+
+# Reporting Violations of this Code of Conduct
+If you believe someone is harassing you or has otherwise violated this Code of Conduct, please contact any of the administrators listed at the top of this document and send us an abuse report. If this is the initial report of a problem, please include as much detail as possible. It is easiest for us to address issues when we have more context.
```

Comment:
```suggestion
If you believe someone is harassing you or has otherwise violated this Code of Conduct, please contact the Passage support team at support@passage.id and send us an abuse report. If this is the initial report of a problem, please include as much detail as possible. It is easiest for us to address issues when we have more context.
```